src/meta/processors/admin/BalanceTask.cpp

```diff
@@ -152,6 +152,14 @@ void BalanceTask::invoke() {
                 if (!resp.ok()) {
                     LOG(INFO) << taskIdStr_ << "Update meta failed, status " << resp;
                     ret_ = Result::FAILED;
+                } else if (kv_ != nullptr) {
+                    if (LastUpdateTimeMan::update(kv_, time::WallClock::fastNowInMilliSec()) !=
```

Comment:
Each task finished will result in the meta info updated?

Reply:
Do you have better suggestion? How about when BalancePlan::invoke is done?

Reply:
Sounds good.